From patchwork Sat Apr  9 07:51:47 2022
Content-Type: text/plain; charset="utf-8"
MIME-Version: 1.0
Content-Transfer-Encoding: 7bit
X-Patchwork-Submitter: Frank Wunderlich <linux@fw-web.de>
X-Patchwork-Id: 12807727
Return-Path: 
 <linux-rockchip-bounces+linux-rockchip=archiver.kernel.org@lists.infradead.org>
X-Spam-Checker-Version: SpamAssassin 3.4.0 (2014-02-07) on
	aws-us-west-2-korg-lkml-1.web.codeaurora.org
Received: from bombadil.infradead.org (bombadil.infradead.org
 [198.137.202.133])
	(using TLSv1.2 with cipher ECDHE-RSA-AES256-GCM-SHA384 (256/256 bits))
	(No client certificate requested)
	by smtp.lore.kernel.org (Postfix) with ESMTPS id 6C355C433F5
	for <linux-rockchip@archiver.kernel.org>;
 Sat,  9 Apr 2022 07:56:36 +0000 (UTC)
DKIM-Signature: v=1; a=rsa-sha256; q=dns/txt; c=relaxed/relaxed;
	d=lists.infradead.org; s=bombadil.20210309; h=Sender:
	Content-Transfer-Encoding:Content-Type:List-Subscribe:List-Help:List-Post:
	List-Archive:List-Unsubscribe:List-Id:MIME-Version:Message-Id:Date:Subject:Cc
	:To:From:Reply-To:Content-ID:Content-Description:Resent-Date:Resent-From:
	Resent-Sender:Resent-To:Resent-Cc:Resent-Message-ID:In-Reply-To:References:
	List-Owner; bh=PSswY+s6PaiyYkKnrnrtgcyMH6V+TJF2U1aoWyx+S7E=; b=HBYLFm+L/O0Yha
	wzDZa58N1OjyMZ1a6BESEo31K+70qDsSbJ9RhC7D1VHBR77gLmOb8aIgeEletMfnh5ABNEI++m49O
	X0bdtyiKDVV/lcRn+VrVQVIg2SMAHvlo3LxVWblEE3i/Nu04Uy27nczY+x3C/PEYAd4HIAJ0YtFdL
	0jxfrx2Q+pkcNwkG95bisF/nPxvGRLzCLrxPmnnYgrbhpQ1Z6rflKmQkAsTqsFP3qFDC5p9eYIF5h
	SWaYx/THQbrPMFQIoyfGx2tEm3huAGOj6Kcpf7uSFCRZPYSfDe0jGIBW2lzsHYLFxh50f4AMCWROJ
	HYOt64c18TsHTWYXCJ8Q==;
Received: from localhost ([::1] helo=bombadil.infradead.org)
	by bombadil.infradead.org with esmtp (Exim 4.94.2 #2 (Red Hat Linux))
	id 1nd5xQ-002Xuf-UX; Sat, 09 Apr 2022 07:56:32 +0000
Received: from mxout2.routing.net ([2a03:2900:1:a::b])
 by bombadil.infradead.org with esmtps (Exim 4.94.2 #2 (Red Hat Linux))
 id 1nd5xF-002Xar-4a; Sat, 09 Apr 2022 07:56:22 +0000
Received: from mxbox1.masterlogin.de (unknown [192.168.10.88])
 by mxout2.routing.net (Postfix) with ESMTP id 6D2A95FDFA;
 Sat,  9 Apr 2022 07:52:10 +0000 (UTC)
DKIM-Signature: v=1; a=rsa-sha256; c=relaxed/relaxed; d=mailerdienst.de;
 s=20200217; t=1649490730;
 h=from:from:reply-to:subject:subject:date:date:message-id:message-id:
 to:to:cc:cc:mime-version:mime-version:
 content-transfer-encoding:content-transfer-encoding;
 bh=Woadcv1V4mB8TNR0hX8FoYo4UZy8QqujR0u1IsxgXzk=;
 b=LotVTLABBbxNHxFz/KdC+RuwH+FBndhUE2ckpJVvYC0M9Oy61s1OBFE1BZcgR16eaav06b
 yEl/OZLRzjecQhNJVU/QnyAUAjeTKaheVVVTLGLtzXADW6o3mSF5KOWUklazdgfPEnO9WD
 sw1XELdCQbyO0Buo2hG2SmbFRLwP4wE=
Received: from localhost.localdomain (fttx-pool-217.61.154.105.bambit.de
 [217.61.154.105])
 by mxbox1.masterlogin.de (Postfix) with ESMTPSA id 848854021D;
 Sat,  9 Apr 2022 07:52:09 +0000 (UTC)
From: Frank Wunderlich <linux@fw-web.de>
To: linux-rockchip@lists.infradead.org
Cc: Frank Wunderlich <frank-w@public-files.de>,
 Rob Herring <robh+dt@kernel.org>, Krzysztof Kozlowski <krzk+dt@kernel.org>,
 Heiko Stuebner <heiko@sntech.de>, Peter Geis <pgwipeout@gmail.com>,
 Michael Riesch <michael.riesch@wolfvision.net>, devicetree@vger.kernel.org,
 linux-arm-kernel@lists.infradead.org, linux-kernel@vger.kernel.org
Subject: [PATCH] arm64: dts: rockchip: Fix clocks for rk356x usb
Date: Sat,  9 Apr 2022 09:51:47 +0200
Message-Id: <20220409075147.136187-1-linux@fw-web.de>
X-Mailer: git-send-email 2.25.1
MIME-Version: 1.0
X-Mail-ID: b784876b-6e21-42de-8301-4411050e2401
X-CRM114-Version: 20100106-BlameMichelson ( TRE 0.8.0 (BSD) ) MR-646709E3 
X-CRM114-CacheID: sfid-20220409_005621_356586_0CE97762 
X-CRM114-Status: UNSURE (   9.82  )
X-CRM114-Notice: Please train this message.
X-BeenThere: linux-rockchip@lists.infradead.org
X-Mailman-Version: 2.1.34
Precedence: list
List-Id: Upstream kernel work for Rockchip platforms
 <linux-rockchip.lists.infradead.org>
List-Unsubscribe: <http://lists.infradead.org/mailman/options/linux-rockchip>,
 <mailto:linux-rockchip-request@lists.infradead.org?subject=unsubscribe>
List-Archive: <http://lists.infradead.org/pipermail/linux-rockchip/>
List-Post: <mailto:linux-rockchip@lists.infradead.org>
List-Help: <mailto:linux-rockchip-request@lists.infradead.org?subject=help>
List-Subscribe: <http://lists.infradead.org/mailman/listinfo/linux-rockchip>,
 <mailto:linux-rockchip-request@lists.infradead.org?subject=subscribe>
Sender: "Linux-rockchip" <linux-rockchip-bounces@lists.infradead.org>
Errors-To: 
 linux-rockchip-bounces+linux-rockchip=archiver.kernel.org@lists.infradead.org

From: Frank Wunderlich <frank-w@public-files.de>

after these 2 commit different clock names are needed compared to 5.17

commit 5114c3ee2487 ("usb: dwc3: Calculate REFCLKPER based on reference clock")
commit 33fb697ec7e5 ("usb: dwc3: Get clocks individually")

change them in new rk356x usb support

Signed-off-by: Frank Wunderlich <frank-w@public-files.de>
Signed-off-by: ....
---
this is a fix for this series not yet applied
https://patchwork.kernel.org/project/linux-rockchip/list/?series=630470
@peter
after testing you can squash it into your series and add a co-developed or similar
---
 arch/arm64/boot/dts/rockchip/rk356x.dtsi | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk356x.dtsi b/arch/arm64/boot/dts/rockchip/rk356x.dtsi
index 55e6dcb948cc..6dfe54e53be1 100644
--- a/arch/arm64/boot/dts/rockchip/rk356x.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk356x.dtsi
@@ -264,8 +264,8 @@ usb_host0_xhci: usb@fcc00000 {
 		interrupts = <GIC_SPI 169 IRQ_TYPE_LEVEL_HIGH>;
 		clocks = <&cru CLK_USB3OTG0_REF>, <&cru CLK_USB3OTG0_SUSPEND>,
 			 <&cru ACLK_USB3OTG0>;
-		clock-names = "ref_clk", "suspend_clk",
-			      "bus_clk";
+		clock-names = "ref", "suspend_clk",
+			      "bus_early";
 		dr_mode = "host";
 		phy_type = "utmi_wide";
 		power-domains = <&power RK3568_PD_PIPE>;
@@ -280,8 +280,8 @@ usb_host1_xhci: usb@fd000000 {
 		interrupts = <GIC_SPI 170 IRQ_TYPE_LEVEL_HIGH>;
 		clocks = <&cru CLK_USB3OTG1_REF>, <&cru CLK_USB3OTG1_SUSPEND>,
 			 <&cru ACLK_USB3OTG1>;
-		clock-names = "ref_clk", "suspend_clk",
-			      "bus_clk";
+		clock-names = "ref", "suspend_clk",
+			      "bus_early";
 		dr_mode = "host";
 		phys = <&usb2phy0_host>, <&combphy1 PHY_TYPE_USB3>;
 		phy-names = "usb2-phy", "usb3-phy";
